@uportal/content-carousel/src/components/ContentCarousel.html

```diff
@@ -8,7 +8,7 @@
   </template>
   <template v-else>
     <div class="slick-inner">
-      <slick ref="slick" :options="computedSlickOptions">
+      <slick ref="slick" :options="computedSlickOptions" v-bind:style="{ height }">
```

Comment:
```suggestion
      <slick ref="slick" :options="computedSlickOptions" :style="{ height }">
```

https://vuejs.org/v2/guide/syntax.html#v-bind-Shorthand